package.json

```diff
@@ -41,7 +41,8 @@
   },
   "peerDependencies": {
     "react": ">=16.13.1",
-    "react-dom": ">=16.13.1"
+    "react-dom": ">=16.13.1",
+    "rxjs": "6.6.2"
```

Comment:
Not sure, that it's a good place for it

Reply:
Perhaps this is more a devDependency... 
please fix it :)